### DISCUSSION TITLE
New API to get available storage space and current usage

### DISCUSSION THREAD

**Discussion by bravadomizzou** [react-native-async-storage/async-storage#1125] [Ideas]
It would be nice if there was an API were you can get how much data is currently stored. Since there is a 6mb limit on Android, it would be nice if there was a way to see what the current usage is and how much space is left.

**Comment by krizzu** [comment]
You can lift current limit on old implementation by [adding a property to the project](https://react-native-async-storage.github.io/async-storage/docs/advanced/db_size/). Alternatively you can use [next storage implementation, which does not have a limit](https://react-native-async-storage.github.io/async-storage/docs/advanced/next)
